assignment1/fc_net.py

```diff
@@ -80,8 +80,24 @@ def L_model_backward(self, AL, Y, caches):
 
         return grads
 
+    def update_parameters(self, parameters, grads, learning_rate):
+        """
+        Updates parameters using gradient descen
+
+        :param parameters: a python dictionary containing the DNN architecture’s parameters
+        :param grads: a python dictionary containing the gradients (generated by L_model_backward)
+        :param learning_rate: the learning rate used to update the parameters (the “alpha”)
+        :return: – the updated values of the parameters object provided as input
+        """
+
+        for layer_idx in range(self.num_layers):
+            old_W, dW = parameters['W' + str(layer_idx)], grads['db' + str(layer_idx)]
```

Comment:
layers idx start from 1. should be layer_idx + 1

Reply:
in other places we started with 0